src/sentry/api/endpoints/organization_integration_repository_project_path_config_details.py

```diff
@@ -1,44 +1,54 @@
 from django.http import Http404
 
 from rest_framework import status
+from rest_framework.response import Response
 
-from sentry.api.bases.organization import OrganizationIntegrationsPermission
-from sentry.api.bases.organization_integrations import OrganizationIntegrationBaseEndpoint
+from sentry.api.bases.organization import OrganizationIntegrationsPermission, OrganizationEndpoint
 from sentry.api.serializers import serialize
 from sentry.models import RepositoryProjectPathConfig
 
 from .organization_integration_repository_project_path_configs import (
     RepositoryProjectPathConfigSerializer,
+    NullableOrganizationIntegrationMixin,
 )
 
 
 class OrganizationIntegrationRepositoryProjectPathConfigDetailsEndpoint(
-    OrganizationIntegrationBaseEndpoint
+    OrganizationEndpoint, NullableOrganizationIntegrationMixin
 ):
     permission_classes = (OrganizationIntegrationsPermission,)
 
-    def convert_args(self, request, organization_slug, integration_id, config_id, *args, **kwargs):
-        args, kwargs = super().convert_args(
-            request, organization_slug, integration_id, config_id, *args, **kwargs
-        )
-
-        org_integration = self.get_organization_integration(kwargs["organization"], integration_id)
-        kwargs["org_integration"] = org_integration
+    def convert_args(self, request, organization_slug, config_id, *args, **kwargs):
+        args, kwargs = super().convert_args(request, organization_slug, config_id, *args, **kwargs)
 
         try:
-            kwargs["config"] = RepositoryProjectPathConfig.objects.get(
+            kwargs["config"] = RepositoryProjectPathConfig.objects.all().get(
```

Comment:
@NisanthanNanthakumar why do we need the `all()` here? I think that's going to query ALL `RepositoryProjectPathConfig` objects which could be expensive?